core/handlers/library/registry.go

```diff
@@ -73,6 +75,8 @@ func InitRegistry(c Config) Registry {
 
 // loadHandlers loads the configured handlers
 func (r *registry) loadHandlers(c Config) {
+	r.authenticationTimeWindow = c.AuthenticationTimeWindow
```

Comment:
I think that `InitRegistry()` is the right place to initialize it. Not here.

Reply:
fix